sunpy/instr/fermi.py

```diff
@@ -40,59 +40,39 @@ def download_weekly_pointing_file(date):
     # use a temp directory to hold the file
     tmp_dir = tempfile.mkdtemp()
     # use Fermi data server to access weekly LAT pointing file.
-    base_url = 'http://fermi.gsfc.nasa.gov/ssc/observations/timeline/ft2/files/'
-    fbasename='FERMI_POINTING_FINAL_'
-
+    base_url = 'ftp://legacy.gsfc.nasa.gov/FTP/glast/data/lat/weekly/spacecraft/'
+    fbasename = 'lat_spacecraft_weekly_w'
+    
     # find out which file to get based on date
-    # earliest file in the FERMI server is for mission week 23,
-    # beginning 2008 November 6.
-    weekly_file_start=parse_time('2008-11-06')
-    base_week=23
+    # earliest full file in the FERMI server is for mission week 10,
+    # beginning 2008 August 7.
+    weekly_file_start=parse_time('2008-08-07')
+    base_week=10
 
     # find out which mission week corresponds to date
     time_diff=date-weekly_file_start
     weekdiff = time_diff.days//7
     week = weekdiff + base_week
-
-    # find out the rest of the file name. Need the year and the day-in-year for
-    # start and end of file
-    start_date = weekly_file_start + datetime.timedelta(weekdiff*7)
-    start_year_str = str(start_date.year) + '-01-01'
-    start_str = start_date.strftime('%Y%j')
-
-    # now end string
-    end_date = weekly_file_start + datetime.timedelta((weekdiff+1)*7)
-    end_str = end_date.strftime('%Y%j')
+    weekstr = ('%03.0f' % week)
 
     # construct the full url for the weekly pointing file
-    full_fname_start=fbasename + str(week) + '_' + start_str + '_' + end_str + '_'
-    full_fname_extension='.fits'
-    # the full filename will be full_fname_start + version number + full_fname
-    # extension, but version number unknown - multiple versions may exist for
-    # each week.
-
-    # search through version numbers starting from most recent (10) until we
-    # find a file. This will find the most up to date file.
-    for v in range(10, -1, -1):
-        rest_of_filename = full_fname_start + '0' + str(v) + full_fname_extension
-        full_fname = base_url + rest_of_filename  # full_fname_start + '0' + str(v) + full_fname_extension
-        try:
-            resp = urllib2.urlopen(full_fname)
-            exists = True
-        except:
-            urllib2.HTTPError
-            exists = False
-        # if the file exists then exit and retain this filepath
-        if exists:
-            break
+    full_fname = fbasename + weekstr + '_p202_v001.fits'
+
+    #try to download the file from the FTP site
+    try:
+        resp = urllib2.urlopen(base_url + full_fname)
+        exists = True
+    except:
+        urllib2.HTTPError
+        exists = False
 
     # if no matches at all were found, then the pointing file doesn't exist
     if not exists:
         raise ValueError('No Fermi pointing files found for given date!')
 
     # download the file
-    pointing_file_url=full_fname
-    destination=os.path.join(tmp_dir,rest_of_filename)
+    pointing_file_url=base_url + full_fname
```

Comment:
if you define this `pointing_file` before line 63, then you can use that variable in 63 instead of defining the url again